templates/zerver/help/change-your-language.md

```diff
@@ -1,37 +1,42 @@
 # Change your language
 
-By default, the Zulip UI is displayed in English.  However, Zulip has
-full support for internationalization, and the UI has been translated
-into many other languages.  You can see the Zulip UI in other
-languages temporarily by modifying the URLs using the appropriate
-language code, e.g. `/ru/` for Russian.
+By default, the Zulip UI is displayed in English.  However, Zulip has full
+support for internationalization. You can see which languages Zulip supports,
+and help add support for new languages by visiting
```

Comment:
Two things:
* Once we get to the link, there might be confusion over what "40%" translated means. Does it mean you can use that language at all? Ideally we would have a succinct explanation of it (thought that might be hard, so fine to add it to a todo list for a later pass)
* It seems weird to claim "which language zulip supports", and then get to a page where there are no languages at 100%. So weakened the language a bit, though perhaps should have gone farther (or better explained that N% translated is actually mostly translated in practical experience).